### PR TITLE
Amp/pool nav

### PR DESCRIPTION
**Pull request by songwongtp**
## Describe your changes
- Add Amp for Pool List page

### CODE REVIEW

File: src/lib/services/amplitude.tsx [hunk 1]
```diff
@@ -209,10 +218,18 @@ export const AmpTrackCopier = (section: Option<string>, type: string) =>
...
   action: "expand" | "collapse",
-  component: "assets" | "json" | "permission_address" | "event_box",
+  component:
+    | "assets"
+    | "json"
+    | "permission_address"
+    | "event_box"
+    | { type: "unsupported_pool"; id: number },
   section: Option<string>
 ) => track(AmpEvent.USE_EXPAND, { action, component, section });
```
Comment (poomthiti) [inline src/lib/services/amplitude.tsx:228]: 1. receive props as object 
```ts 
AmpTrackExpand = ({action, component,...}: interface) => {}
```
2. just | "unsupported_pool" and receive another prop (info: object)

**Review by poomthiti — CHANGES_REQUESTED** [review]

File: src/lib/pages/pools/components/supportedSection/index.tsx [hunk 2]
```diff
@@ -172,15 +179,19 @@ export const SupportedSection = ({
...
               Sort Pool ID:
             </Text>
             <Button
               variant="outline-gray"
               size="sm"
-              pr="1"
-              onClick={() => setShowNewest(!showNewest)}
+              pr={1}
+              onClick={() => {
+                const newOrdering = !showNewest;
```
Comment (bkioshn) [inline src/lib/pages/pools/components/supportedSection/index.tsx:191]: let's name this `isDesc` so it is the same as unsupported page

File: src/lib/pages/pools/components/unsupportedSection/index.tsx [hunk 3]
```diff
@@ -143,41 +153,38 @@ export const UnsupportedSection = ({
...
               variant="outline-gray"
               w="94px"
               size="sm"
               onClick={() =>
-                setExpandedIndexes((prev) =>
-                  prev.length ? [] : Array.from(Array(pageSize).keys())
-                )
+                setExpandedIndexes((prev) => {
+                  const isExpand = !prev.length;
+                  AmpTrackExpandAll(isExpand ? "expand" : "collapse");
```
Comment (poomthiti) [inline src/lib/pages/pools/components/unsupportedSection/index.tsx:185]: personally think this should be outside of setState fn

**Review by poomthiti — CHANGES_REQUESTED** [review]

File: src/lib/services/amplitude.tsx [hunk 4]
```diff
@@ -229,3 +254,12 @@ export const AmpTrackPaginationNavigate = (
     pageSize,
     currentPage,
   });
+
+export const AmpTrackUseSort = (order: "Ascending" | "Descending") =>
```
Comment (bkioshn) [inline src/lib/services/amplitude.tsx:258]: Should the union type be uncapitalize like the above? just for consistency

File: src/lib/pages/pools/components/unsupportedSection/index.tsx [hunk 5]
```diff
@@ -143,42 +153,42 @@ export const UnsupportedSection = ({
...
-                )
-              }
+              onClick={() => {
+                AmpTrackExpandAll(
+                  !expandedIndexes.length ? "expand" : "collapse"
+                );
+                setExpandedIndexes(
+                  !expandedIndexes.length
+                    ? Array.from(Array(pageSize).keys())
+                    : []
```
Comment (poomthiti) [inline src/lib/pages/pools/components/unsupportedSection/index.tsx:189]: call (prev) => ... for this

**Review by poomthiti — CHANGES_REQUESTED** [review]

**Review by poomthiti — APPROVED** [review]

**Review by bkioshn — APPROVED** [review]

**Pull request merged**